src/masonite/utils/location.py

```diff
@@ -0,0 +1,44 @@
+"""Helpers to resolve absolute paths to the different app resources using a configured
+location."""
+from os.path import join, abspath
+
+
+def _normalize_path(path):
+    """Transform dotted path to normal file path."""
+    return path.replace(".", "/")
+
+
+def _build_path(location_key, relative_path, absolute):
+    from wsgi import application
```

Comment:
should we hard code this here? I'm hesitant on hard coding location paths on something that is used by the developer

Reply:
I am not sure of you mean by this. the idea behind this PR is to be avoid hard-coding paths, ensure consistency throughout the codebase and avoid hard-coding path building
I will clarify the idea and how/when to use this in next comment 😉

Reply:
Right but what if the application is not in the wsgi.py file? should we require at least this file?

Reply:
ah..was thinking that this was always there

If we look at the dynamic configuration of views in here
```python
class Kernel:
    # ....
    def register_templates(self):
        self.application.bind("views.location", "tests/integrations/templates")
```
This makes sense and can be used only if there is an application with this Kernel registered. As is I feel that the wsgi.py file defined as it is now is already a requirement for the project, but maybe that's the only one as almost all other parts are built dynamically and can be swapped out or removed.